.travis.yml

```diff
@@ -1,20 +1,20 @@
 language: python
 python:
-- 2.7
-- 3.4
-notifications:
-email: false
-
-before_install:
-  pip install codecov 
+  - 2.7
+  - 3.4
+  - 3.5
+  - 3.6
+cache: pip
+before_cache:
+  - rm -f $HOME/.cache/pip/log/debug.log
 install:
-- pip install -U setuptools
-- pip install nose
-- pip install mock
-- pip install coverage
+  - pip install -U setuptools
```

Comment:
nit: did you try if we (still) need this?

Reply:
Doesn't seem to be needed in your csvw config. So probably not.